app/api/orders.py

```diff
@@ -61,6 +61,10 @@ def before_post(self, args, kwargs, data=None):
         if not has_access('is_coorganizer', event_id=data['event']):
             data['status'] = 'pending'
 
+        # Setting default value of amount in possible error cases
+        if ('amount' not in data) or (data['amount'] is None) or (not data.get('amount')):
+            data['amount'] = 0
```

Comment:
Why can't marshmallow schema handle this?

Reply:
@iamareebjamal it can handle it. Thanks for pointing out, applying the required changes